coordinator/quote/ert.go

```diff
@@ -26,6 +26,8 @@ type PackageProperties struct {
 	ProductID *uint64
 	// Security version number of the package.
 	SecurityVersion *uint
+	// Accepted TCB levels
+	AcceptedTCBLevels []string
```

Comment:
I wonder if we should define this per package or globally in the manifest. I don't have a strong opinion. What do you think? @OverOrion, @daniel-weisse, @m1ghtym0

Reply:
I think having the option to define this per package makes sense, since applications might be running on different systems (but are verified by the same instance of MarbleRun). There could be applications where a certain TCB level is acceptable, which might not be the case for other applications.

Reply:
This is what I had in mind as well.

Reply:
Yes, I agree.